admin/js/boldgrid-backup-admin-settings-autoupdate.js

```diff
@@ -53,7 +53,9 @@ BOLDGRID.SETTINGS = BOLDGRID.SETTINGS || {};
 			$bgBox
 				.find( '.table-help td p' )
 				.attr( 'style', 'height: 0em; opacity: 0%; position: relative; z-index:-1' );
-			$bgBox.find( '.dashicons-editor-help, .help-icon' ).on( 'click', self._toggleHelp );
+			$bgBox
```

Comment:
Is this change related to the issue? Did you intend to add this?

Reply:
This is to resolve a different but related bug that there was no issue for. The help icons were not working in some places as a result of my code for the ones within the table body being done differently. I had to increase the specificity of the selector for it, so that my code wasn't effecting all the other help icons on the settings page.